zng/docs/spec.md

```diff
@@ -511,15 +538,16 @@ An application-specific payload has the following form:
 Here, `<control code>` is a decimal integer in the range 6-127 and `<payload>`
 is any UTF-8 string with escaped newlines.
 
-### Type Grammar
+### 4.2 Type Grammar
```

Comment:
I don't know if it was intentional or an oversight that this section didn't have a number before. However, once there's a TOC, it's usually populated with the headings that make up the sections, and once we start numbering sections it feels odds to have non-numbered, heading-labeled sections, so it seemed appropriate to add one here.